config/locales/en.yml

```diff
@@ -111,7 +113,10 @@ en:
         number_of_children: Enter the number of children
       steps_help_with_fees_help_paying_form:
         help_paying:
-          <<: *YESNO
+          yes_with_ref_number: Yes, I have a Help with Fees reference number
+          yes_without_ref_number: Yes, I don’t have a Help with Fees reference number
```

Comment:
Please change this to; `Yes, but I don’t have a Help with Fees reference number`
I think that's a lot clearer, in context. I have run this past Dave H.

Reply:
Ok, I was not aware of that. I will change it.